docs/user/guides/setup/git-repo-manual.rst

```diff
@@ -29,11 +28,11 @@ This includes most Git providers, for example |git_providers_and|.
 .. _webhook-integration-bitbucket:
 .. _webhook-integration-gitlab:
 
-Provider-specific instructions
-------------------------------
+Manual integration setup
+------------------------
 
-You need to configure your Git provider to call a webhook on Read the Docs.
-This will make Read the Docs build your documentation when a new commit, branch or tag is pushed to your repository.
+You need to configure your Git provider integration to call a webhook that alerts Read the Docs of changes.
+Read the Docs will sync versions, build your documentation, and other action when your Git repository is updated.
```

Comment:
This is an improvement, though `and other action` seems okay to drop if we aren't defining anything else here:

```suggestion
Read the Docs will sync versions and build your documentation when your Git repository is updated.
```

We could also get specific with what happens here, though it's maybe on the background side of things.